src/editor.c

```diff
@@ -1907,7 +1967,7 @@ static gchar *find_calltip(const gchar *word, GeanyFiletype *ft)
 	g_return_val_if_fail(ft && word && *word, NULL);
 
 	/* use all types in case language uses wrong tag type e.g. python "members" instead of "methods" */
-	tags = tm_workspace_find(word, NULL, tm_tag_max_t, NULL, ft->lang);
+	tags = tm_workspace_find(word, strlen(scope)==0?NULL:scope, tm_tag_max_t, NULL, ft->lang);
```

Comment:
Need to change tm_workspace_find function too I just realized. Currently this will prevent calltips on objects e.g.

ClassA { static func(abc); }

ClassA.func(abc); will complete correctly
but now:
ClassA aObj ();
aObj.func() will no longer complete because when tm_workspace_find is passed a non-null scope it checks every calltips scope against it and hides the ones which aren't equal